guide/01-getting-started/system-requirements.ipynb

```diff
@@ -22,44 +22,57 @@
    "metadata": {},
    "source": [
     "## Operating System \n",
-    "The ArcGIS API for Python is compatible with 32-bit and 64-bit versions of Windows, macOS, and Linux.\n",
+    "The ArcGIS API for Python 2.2.0 is compatible with 64-bit versions of Windows, macOS, and Linux. Note that, 32-bit versions of Windows and Linux is no longer supported for API versions later than 1.7.1.\n",
     "\n",
     "## Python Version\n",
-    "Python 3.5 or later is required to use the ArcGIS API for Python.\n",
+    "Python 3.9.x to 3.11.x is required to use the ArcGIS API for Python 2.2.0.\n",
+    "\n",
+    "> **Note:** The `arcgis.learn` module is not supported with Python 3.11.x.\n",
```

Comment:
or 3.10 as of currently.  Maybe rephrase as arcgis.learn is only supported on 3.9?

Reply:
Done!